Jenkinsfile

```diff
@@ -10,9 +10,9 @@ node {
   docker.image('cloudbees/java-build-tools').inside {
     withEnv(['GIT_COMMITTER_EMAIL=me@hatescake.com','GIT_COMMITTER_NAME=Hates','GIT_AUTHOR_NAME=Cake','GIT_AUTHOR_EMAIL=hates@cake.com']) {
       try {
+        sh "node ./bin/checkshrinkwrap.js"
         sh "mvn clean install -B -DcleanNode -Dmaven.test.failure.ignore"
         sh "node ./bin/checkdeps.js"
```

Comment:
@scherler no - that actually breaks as there is no node_modules yet - it has to run after the build